src/api/QdmCalculationService.ts

```diff
@@ -133,6 +134,29 @@ export class QdmCalculationService {
     return groupPass;
   }
 
+  mapObservations = (population, results) => {
```

Comment:
seems specific to patient-based. can we rename it to make look specific to patient-based?

Reply:
I have changed it to "mapPatientBasedObservations"  to make it more specific